node_modules/oae-core/userdetails/userdetails.html

```diff
@@ -0,0 +1,63 @@
+<!-- CSS -->
+<link rel="stylesheet" type="text/css" href="css/userdetails.css" />
+
+<!-- MODAL -->
+<div id="userdetails-modal" class="modal fade" tabindex="-1" aria-labelledby="userdetails-modal-title" aria-hidden="true">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal" aria-label="__MSG__CLOSE__">
+                    <span aria-hidden="true">&#215;</span>
+                </button>
+                <h3 id="userdetails-modal-title">__MSG__USER_DETAILS__</h3>
+            </div>
+            <div class="modal-body">
```

Comment:
The modal can be restyled to look like this:

![screen shot 2015-05-22 at 08 04 43](https://cloud.githubusercontent.com/assets/109850/7819634/e870f80e-0397-11e5-8234-1591bb59d30e.png)